lib/game.rb

```diff
@@ -0,0 +1,176 @@
+module PlanetExpress
+  class Game
+
+    def initialize
+      @bender = PlanetExpress::Bender.new(self)
+      @fry = PlanetExpress::Fry.new(self)
+      @hermes = PlanetExpress::Hermes.new(self)
+      @leela = PlanetExpress::Leela.new(self)
+      @zoidberg = PlanetExpress::Zoidberg.new(self)
+      @stable = true
+    end
+
+    def play_game
+      loop do
+        @bender = PlanetExpress::Bender.new(self)
+        @fry = PlanetExpress::Fry.new(self)
+        @hermes = PlanetExpress::Hermes.new(self)
+        @leela = PlanetExpress::Leela.new(self)
+        @zoidberg = PlanetExpress::Zoidberg.new(self)
+        @stable = true
+        do_play_game
+        puts "Do you want to play again? (y/n)"
+        break unless gets.chomp == "y"
+      end
+    end
+
+    def do_play_game
+      check
+      while @stable == true
+        prompt
+      end
+    end
+
+    def make_choice
+      check
+      while @stable == true
+        i = gets.chomp.to_i
+        if i == 1
+          steal
+        elsif i == 2
+          drink
+        elsif i == 3
+          account
+        elsif i == 4
+          deliver
+        elsif i == 5
+          hunger
+        end
+        prompt
+      end
+    end
+
+    def score
+      b = @bender.horde
+      f = @fry.thirst
+      h = @hermes.receipts
+      l = @leela.work
+      z = @zoidberg.hunger
+      puts "Crew Status:"
+      puts "Bender has #{b} items in his horde."
+      puts "Fry has #{f} units of thirst."
+      puts "Hermes has #{h} receipts to enter."
+      puts "Leela has #{l} unfulfilled work orders."
+      puts "Zoidberg has #{z} units of hunger."
+      total_score = b + f + h + l + z
+      puts "This is your total score: #{total_score}"
+    end
+
+    def drink
+      @fry.thirst -= 10
+      @leela.work += 3
+
+      if @fry.thirst < 0
```

Comment:
I think it would be better if you would put this conditional around the `-=` and `+=` to prevent this case from happening in the first place